tools/swift-inspect/Sources/swift-inspect/Operations/DumpConcurrency.swift

```diff
@@ -76,8 +77,8 @@ fileprivate class ConcurrencyDumper {
     self.process = process
 
     func getMetadata(symbolName: String) -> swift_reflection_ptr_t? {
-      let addr = process.GetSymbolAddress(symbolName)
-      if let ptr = process.ReadBytes(addr, MemoryLayout<UInt>.size) {
+      let addr = process.getAddr(symbolName: symbolName)
```

Comment:
I think that this change is less desirable - the `GetSymbolAddress` is the generic entry that works across all targets.